concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+Lastra-1986-381-45	45	barriga	belly	1251	BELLY
+Lastra-1986-381-46	46	ombligo (sin posesivo)	navel (without possessive)	1838	NAVEL
+Lastra-1986-381-47	47	nalgas	buttocks	834	BUTTOCKS
+Lastra-1986-381-48	48	mano (sin posesivo)	hand (without possessive)	1277	HAND
+Lastra-1986-381-49	49	tu mano	your hand		
+Lastra-1986-381-50	50	uña (sin posesivo)	NAIL (without possessive)	1896	FINGERNAIL OR TOENAIL
+Lastra-1986-381-51	51	mi uña	my nail		
+Lastra-1986-381-52	52	codo	elbow	981	ELBOW
+Lastra-1986-381-53	53	pierna (especificar de dónde a dónde la llaman así)	leg (specify from where to where they called it like this)	1297	LEG
+Lastra-1986-381-54	54	rodilla (especificar si distinguen la 'rótula')	knee (specify whether they distinguish the ""kneecap"")	1371	KNEE
+Lastra-1986-381-55	55	mi pie	my foot	1301	FOOT
+Lastra-1986-381-56	56	corazón	heart	1223	HEART
+Lastra-1986-381-57	57	hígado	liver	1224	LIVER
+Lastra-1986-381-58	58	mi hueso	My bone	1394	BONE
+Lastra-1986-381-59	59	piel (de una persona)	skin (of a person)	2613	SKIN (HUMAN)
+Lastra-1986-381-60	60	sangre	blood	946	BLOOD
+Lastra-1986-381-61	61	mi sangre	my blood		
+Lastra-1986-381-62	62	saliva	saliva	1503	SALIVA
+Lastra-1986-381-63	63	lágrima	tear	1651	TEAR (OF EYE)
+Lastra-1986-381-64	64	pus	pus	1808	PUS
+Lastra-1986-381-65	65	lagaña	lagaña		
+Lastra-1986-381-66	66	caspa	dandruff	576	DANDRUFF
+Lastra-1986-381-67	67	sarna	scabies		
+Lastra-1986-381-68	68	viruela	smallpox		
+Lastra-1986-381-69	69	su cuerno	his horn	1393	HORN (ANATOMY)
+Lastra-1986-381-70	70	pico (de un pájaro)	peak (of a bird)	73	BEAK
+Lastra-1986-381-71	71	ala	wing	1257	WING
+Lastra-1986-381-72	72	pluma	feather	1201	FEATHER
+Lastra-1986-381-73	73	molleja	gizzard		
+Lastra-1986-381-74	74	huevo	egg	744	EGG
+Lastra-1986-381-75	75	mujer	women	962	WOMAN
+Lastra-1986-381-76	76	mi esposa	my wife	1199	WIFE
+Lastra-1986-381-77	77	hombre	man	1554	MAN
+Lastra-1986-381-78	78	marido	husband	1200	HUSBAND
+Lastra-1986-381-79	79	papá	dad	1217	FATHER
+Lastra-1986-381-80	80	mamá	mother	1216	MOTHER
+Lastra-1986-381-81	81	mi hijo	my son		
+Lastra-1986-381-82	82	huérfano	orphan	1149	ORPHAN
+Lastra-1986-381-83	83	entenado (o hijastro)	stepson		
+Lastra-1986-381-84	84	padrastro	stepfather	237	STEPFATHER
+Lastra-1986-381-85	85	madrastra	stepmother	236	STEPMOTHER
+Lastra-1986-381-86	86	curandero	medicine man	394	SHAMAN (FOLK HEALER)
+Lastra-1986-381-87	87	brujo	sorcerer	2137	MAGICIAN
+Lastra-1986-381-88	88	animal (ver si hay differencia entre animal pequeño o grande, doméstico o salvaje)	Animal (see if there is differented between small or large, domestic or wild animal)	619	ANIMAL
+Lastra-1986-381-89	89	guajolote (si además distiguen por tamaño y sexo, anotar enfrente)	Guajolote (if they also distribute by size and sex, write down in front)	2386	TURKEY
+Lastra-1986-381-90	90	gallina	hen	1514	HEN
+Lastra-1986-381-91	91	gavilán	hawk	1865	BUZZARD
+Lastra-1986-381-92	92	zopilote	buzzard		
+Lastra-1986-381-93	93	tecolotes	Tecolotes		
```

Comment:
Would `735 OWL` work or is this too general of a concept?

Reply:
I believed this concept was too broad. But this is a good question.

Only the species that come from Mexico and the south of USA can be called `tecolote`. However, they are also owls. But most of the species from the owl genera cannot receive this name. The categorisation is not referring to a different order or family or species, then, it's just the name some owls in Mexico and surrounding areas are receiving.

Perhaps, then, the best strategy is to map this as `735 OWL`.